cli/cook/__main__.py

```diff
@@ -36,4 +36,6 @@ def sigint_handler(_, __):
 signal.signal(signal.SIGINT, sigint_handler)
 
 if __name__ == '__main__':
+    import requests
+    http.inject_http_modules(requests, requests.adapters)
```

Comment:
Since these are defaulted in `configure()` anyway, I don't think you need to call this here.

Reply:
Done